report/sections/secondary-groups.tex

```diff
@@ -1,3 +1,25 @@
 \section{Secondary Groups}
 
-\todo{TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group}
+%TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group
+Our Secondary groups were Computer vision and Emotion Synthesis. Our task in final pipeline was to read the input given by CV group and send the output to ES group.
```

Comment:
- [ ] capitalize v in Vision, parentheses after each with their abbreviations
- [ ] 'the' before final
- [ ] "CV" ... "ES" see first point